torch/onnx/_internal/torchscript.py

```diff
@@ -0,0 +1,426 @@
+    env: Dict[_C.Value, _C.Value]
+    # TODO(justinchuby): What should we call env?
+
+    def op(
+        self,
+        opname: str,
+        *raw_args: Union[torch.Tensor, _C.Value],
+        outputs: int = 1,
+        **kwargs,
+    ) -> Union[_C.Value, Tuple[_C.Value, ...]]:
+        """Creates an ONNX operator "opname", taking "args" as inputs and attributes "kwargs".
+
+        The set of operators and the inputs/attributes they take
+        is documented at https://github.com/onnx/onnx/blob/master/docs/Operators.md
+
+        This function is monkey-patched onto Graph.
+
+        Args:
+            g: The Torch graph.
+            opname: The ONNX operator name, e.g., `Abs` or `Add`, or an operator qualified
+                with a namespace, e.g., `aten::add`.
+            raw_args: The inputs to the operator; usually provided
+                as arguments to the `symbolic` definition.
+            outputs: The number of outputs this operator returns.
+                By default an operator is assumed to return a single output.
+                If `outputs` is greater than one, this functions returns a tuple
+                of output `Node`, representing each output of the ONNX operator
+                in positional.
+            kwargs: The attributes of the ONNX operator, whose keys are named
+                according to the following convention: `alpha_f` indicates
+                the `alpha` attribute with type `f`.  The valid type specifiers are
+                `f` (float), `i` (int), `s` (string) or `t` (Tensor).  An attribute
+                specified with type float accepts either a single float, or a
+                list of floats (e.g., you would say `dims_i` for a `dims` attribute
+                that takes a list of integers).
+
+        Returns:
+            The node representing the single output of this operator (see the `outputs`
+            keyword argument for multi-return nodes).
+        """
+        return graph_op(self.graph, opname, *raw_args, outputs=outputs, **kwargs)
+
+    def at(
+        self, operator: str, *args, overload_name: str = "", **kwargs
+    ) -> Union[_C.Value, Tuple[_C.Value, ...]]:
+        return aten_op(
+            self.graph, operator, *args, overload_name=overload_name, **kwargs
+        )
+
+    # Relay methods from _C.Graph for compatibility with symbolic functions that expect
+    # a _C.Graph
+    def __getattr__(self, name: str) -> Any:
+        return getattr(self.graph, name)
+
+
+@_beartype.beartype
+def graph_op(
+    g: _C.Graph,
+    opname: str,
+    *raw_args: Union[torch.Tensor, _C.Value],
+    outputs: int = 1,
+    **kwargs,
+) -> Union[_C.Value, Tuple[_C.Value, ...]]:
+    """Creates an ONNX operator "opname", taking "args" as inputs and attributes "kwargs".
+
+    The set of operators and the inputs/attributes they take
+    is documented at https://github.com/onnx/onnx/blob/master/docs/Operators.md
+
+    This function is monkey-patched onto Graph.
```

Comment:
Why monkey-patching `Graph` and not defining it there?

In code documentation, "why" might be more important than the "what" or "how", given what/how is described by the source-code.

Reply:
This function is moved from _patch_torch unchanged. I will add a comment of why

Reply:
What is the process of getting rid of monkey patching _C.Graph, if that's intended?

Reply:
I think it's the meta internal tests I am concern about. Do you think this will affect them?